tests/test_vertex_nomination_via_SGM.py

```diff
@@ -0,0 +1,102 @@
+import pytest
+import numpy as np
+from graspologic.nominate import VNviaSGM
+
+np.random.seed(0)
+
+
+class TestGMP:
+    def test_VNviaSGM_inputs(self):
+        with pytest.raises(ValueError):
+            VNviaSGM(order_voi_subgraph=-1)
+        with pytest.raises(ValueError):
+            VNviaSGM(order_voi_subgraph=1.5)
+        with pytest.raises(ValueError):
+            VNviaSGM(order_seeds_subgraph=-1)
+        with pytest.raises(ValueError):
+            VNviaSGM(order_seeds_subgraph=1.5)
+        with pytest.raises(ValueError):
+            VNviaSGM(n_init=-1)
+        with pytest.raises(ValueError):
+            VNviaSGM(n_init=1.5)
+
+        with pytest.raises(ValueError):
+            VNviaSGM().fit(
+                np.random.randn(3, 4),
+                np.random.randn(4, 4),
+                0,
+                [np.arange(2), np.arange(2)],
+            )
+        with pytest.raises(ValueError):
+            VNviaSGM().fit(
+                np.random.randn(4, 4),
+                np.random.randn(3, 4),
+                0,
+                [np.arange(2), np.arange(2)],
+            )
+        with pytest.raises(ValueError):
+            VNviaSGM().fit(
+                np.random.randn(4, 4),
+                np.random.randn(4, 4),
+                0,
+                [np.arange(2), np.arange(3)],
+            )
+
+    def _get_AB(self):
```

Comment:
what are these graphs? is there a reason you are hard coding this rather than just sampling an ER graph or something?

Reply:
These were the graphs used to test the GMP when I wrote these tests before it was modified. It probably makes more sense to do a ER graph, so I will change that now.